server/routes/overview/supportNeedsView.ts

```diff
@@ -0,0 +1,21 @@
+import type { PrisonerSummary } from 'viewModels'
+
+export default class SupportNeedsViewView {
```

Comment:
I'm imagining we will add properties to this  view class that contain the prisoner's support needs (ie. those that we get back from Curious, possibly via a DTO or view model object or similar?)